lib/zerossl.ts

```diff
@@ -51,12 +51,14 @@ export class ZeroSSL {
     const qs = this.queryString({ access_key: this.options.accessKey })
     const url = `${this.options.apiUrl}/certificates?${qs}`
 
-    const postFn = superagent.post(url)
+    let postFn = superagent.post(url)
       .type('form')
       .field('certificate_domains', options.domains.join(','))
       .field('certificate_csr', options.csr)
       .field('certificate_validity_days', options.validityDays)
       .field('strict_domains', options.strictDomains)
+    
+    if (options.replacementForCertificate) postFn = postFn.field('replacement_for_certificate', options.replacementForCertificate as string)
```

Comment:
It's not necessary to make `postFn` mutable here, as the superagent function chain works on a mutable request object, rather than returning a new one from each function.

Change it back to `const postFn` and remove `postFn =` here, using the field function without reassignment.